crates/ruff/src/message/json.rs

```diff
@@ -63,6 +63,7 @@ pub(crate) fn message_to_json_value(message: &Message) -> Value {
 
     json!({
         "code": message.kind.rule().noqa_code().to_string(),
+        "code_description": message.kind.rule().url(),
```

Comment:
```suggestion
        "description_url": message.kind.rule().url(),
```
i'd add that this is a link

Reply:
Hmm, torn -- I like adhering to the LSP schema.

Reply:
Oh, actually, it should be like this (which hopefully resolves this):

```ts
/**
 * Structure to capture a description for an error code.
 *
 * @since 3.16.0
 */
export interface [CodeDescription](https://microsoft.github.io/language-server-protocol/specifications/lsp/3.17/specification/#codeDescription) {
	/**
	 * An URI to open with more information about the diagnostic error.
	 */
	href: [URI](https://microsoft.github.io/language-server-protocol/specifications/lsp/3.17/specification/#uri);
}
```

Reply:
Okay, you were right, changed my mind after Micha _also_ brought this up.

Reply:
Apparently Mitte calls this `url` so running with that.